doc/source/tutorials/animations.rst

```diff
@@ -0,0 +1,150 @@
+.. _animations:
+
+.. index:: animations
+
+How to create an animation using Emperor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this tutorial we describe how to create a principal coordinates analysis
+(PCoA) plot, and display animated traces of the samples sorted by a metadata
+category.
+
+To do this, we need to have two metadata categories, a *gradient* category, and
+a *trajectory* category. The *gradient* category determines the order in which
+samples are connected together, the *trajectory* category determines how
+samples are grouped together.
+
+In most cases these metadata columns already exist as part of your sample
+information, however you may need to do some curation to make these compatible
+with Emperor. As an example, in a longitudinal study where you wish to track
+the oral microbiome changes in a cohort of 3 mice over the course of 5 weeks,
+each sample will be described by the following columns:
+
+* ``cage_number``: the cage where each mice was housed, more than one mice could
```

Comment:
AFAIK, you don't need this for the animation, right? But perhaps I need to read further down.